be/src/exec/es/es_scroll_parser.h

```diff
@@ -50,5 +50,7 @@ class ScrollParser {
 
     rapidjson::Document _document_node;
     rapidjson::Value _inner_hits_node;
+
+    bool _use_doc_value;
```

Comment:
maybe use doc_value_mode is more suitable？
in future, we can use different parser for _source or doc_value mode